CurrencyConverterApplication/ViewModel/ProductsViewModel.cs

```diff
@@ -0,0 +1,91 @@
+﻿using CurrencyConverterApplication.Data;
+using CurrencyConverterApplication.Model;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CurrencyConverterApplication.ViewModel
+{
+    public class ProductsViewModel : ViewModelBase
+    {
+       
+        private readonly IProductDataProvider _productDataProvider;
+       
+
+
+        public ProductsViewModel(IProductDataProvider productDataProvider)
+        {
+            _productDataProvider = productDataProvider;
+           
+
+                    
+        }
+        private ObservableCollection<ProductViewItem> products=new ObservableCollection<ProductViewItem>();
+        public ObservableCollection<ProductViewItem> Products { get=>products; set { products = value; } }
+       
+        private string? col3;
+
+        public string? Col3
+        {
+            get => col3;
+            set
+            {
+                col3 = value;                
```

Comment:
Should we raise an event if "if(col3==value)" here?

Reply:
I have removed totally the Col3 as with new adjustments l didnt need it. 
thanks